guides/doc-Content_Management_Guide/topics/QuickStart.adoc

```diff
@@ -0,0 +1,104 @@
+[[Syncing_Centos_Stream]]
+== Syncing/Creating {customcontenttitle}
+This chapter outlines how you can manage Centos Streams {customcontent} of different types to {Project}. We will over the following steps.
+
+* Create `Custom` Repositories to mirror content from the Centos Stream
+* Synchronize the Content
+* Setup `Activation Keys`
+* Register Hosts to consume this content.
+* Manage Hosts, install/update/or remove rpms on `Host` via `katello-agent`
+
+
+== Create repositories to synchronize
+
+As a first step we need to let {Project} discover available repositories in the Centos stream. We then pick the repositories to mirror and bundle them into a Product.
+
+.Procedure
+. In the {Project} web UI, navigate to *Content* > *Products*
+. Click *Repo Discovery*.
+. In the *Repository Type* field, choose *Yum Repositories*.
+. In the *URL to Discover* field, enter the Centos Stream url `http://mirror.centos.org/centos/8-stream`.
+. Click *Discover*.
+. Once the repository discovery process is complete you should be able to list of repositories found under that url.
```

Comment:
```suggestion
```
We don't document steps where the user doesn't do anything, so just removing this :)